update PortionOfMatter definitions

akleinau opened the pull request:
closes #280. 
Makes PortionOfMatter subclass definitions aristotelean.
The following classes remain after I updated the definitions because there whole defs need a rework:
LiquidBiofuel
ManufacturedCoalBasedGas
CollieryGas
OilAndPetroleumProducts
GasDieselOil
Perfluorocarbon
DamnedWater
Ocean
PumpedWater
Wave

akleinau commented:
@stap-m, @l-emele can you help create those defs?

akleinau commented:
maybe we should leave Ocean and Wave for now because I don't think they are good PortionOfMatter subclasses.

l-emele commented:
Some proposals:
* _A liquid biofuel is a biofuel that has normal state of matter liquid._
* _A manufactured coal based gas is a gaseous fossil fuel that is manufactured from coal._
* _Perfluorocarbons (PFCs) are portions of matter consisting of organic compounds that contain fluorine atoms but no hydrogen atoms._ This is in line with the HFC definition. Do we already have the concept _organic_ in the OEO? If not, could we import that from ChEBI? @jannahastings 
* _Colliery gas is a fossil gaseous fuel found in coal mines (British English: colliery). The main component is methane. Synonyms are firedamp, mine gas._
* _Pumped water is water which was pumped into an upper reservoir and thus contains potential energy._

akleinau commented:
than there are just missing:
GasDieselOil
OilAndPetroleumProducts
DamnedWater

I'll make this a "real" pull request so the other adapted definitions can be reviewed too.

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -718,7 +718,7 @@ Biofuels can be derived directly from plants (i.e. energy crops), or indirectly
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Biogas is a portion of matter which has a gaseous state and is composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Biogas is a gaseous biofuel and is composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
```

akleinau replied:
Same as charcoal, biogas is primarily a subclass of portion of matter. Would it be enough to add another sentence like "It is used as a fuel" at the end?

stap-m replied:
Actually, for biogas it could make sense to make it a direct subclass of biofuel. I think the term is primarily used in the fuel context, unlike e.g. methane.

akleinau replied:
ok it is now a direct subclass of Biofuel with the corresponding definition

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -718,7 +718,7 @@ Biofuels can be derived directly from plants (i.e. energy crops), or indirectly
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Biogas is a portion of matter which has a gaseous state and is composed principally of methane and carbon dioxide produced by anaerobic digestion of biomass.",
         <http://purl.obolibrary.org/obo/IAO_0000119> "REGULATION (EC) No 1099/2008 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 22 October 2008 on energy statistics https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32008R1099&from=EN"
```
```suggestion

```
If we deviated so strongly from the source we should delete it.

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -810,7 +810,7 @@ Class: CarbonDioxideEmission
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "The solid residue of the destructive distillation and pyrolysis of wood and other vegetal material."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Charcoal is a portion of matter consisting of the solid residue of the destructive distillation and pyrolysis of wood and other vegetal material."@en,
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Charcoal is a fuel consisting of the solid residue of the destructive distillation and pyrolysis of wood and other vegetal material."@en,
```

akleinau replied:
Charcoal is primarily a subclass of portion of matter. Would it be enough to add another sentence like "It is used as a fuel" at the end?

akleinau commented:
I updated the definitions in a way that they are coherent with the asserted hierarchy. I added the use as fuel as another sentence like I wrote with charcoal and biogas, also for the new definitions of @l-emele. That way the definitions follow the asserted view but also incorporate the inferred view with the new fuel hierarchy.

akleinau commented:
OilAndPetroleumProducts: "OilAndPetroleumProducts are portions of matter derived from crude oil (petroleum) as it is processed in oil refineries." (source: https://en.wikipedia.org/wiki/Petroleum_product)
DamnedWater: "Damned water is water stopped by dams from flowing and thus contains energy useable for hydropower."
